### ISSUE TITLE
Mapping is ambiguous

### ISSUE THREAD

**Issue by glmars** [bfil/scala-automapper#6]
I have such compiler error:
```scala
Error:(64, 85) reference to Mapping is ambiguous;
it is imported twice in the same scope by
import io.bfil.automapper._
and import org.springframework.web.bind.annotation._
	private def map(source: Message): MessageDTO = automap(source).to[MessageDTO]
```

if there are two imports in scope:
```scala
import org.springframework.web.bind.annotation._
import io.bfil.automapper._
```

I suppose, because of mapping generation doesn't use fully qualified name for `Mapping`, please, see:
https://github.com/bfil/scala-automapper/blob/fce19bf4c32cdac201a73acf45d497e70ddb12e2/automapper-macros/src/main/scala/io/bfil/automapper/Mapping.scala#L165-L171

**Comment by bfil** [comment]
Thanks for raising the issue, it makes a lot of sense to have the fully-qualified name.

I've released a new version with the fix: **0.6.1**.

Let me know if it solves your problem.

**Comment by glmars** [comment]
I just tested new version (0.6.1), the problem was fixed! Excellent release! Thank you.
